README.md

```diff
@@ -188,20 +194,16 @@ The local plugin inside [ssb-server](https://github.com/ssbc/ssb-server) will us
 ```js
 {
   gossip: {
-    connections: 3, // this one doesn't work any more
     local: true,
     friends: true,
     global: true
   }
 }
 ```
-
-~~Set the max number of gossip connections to sustain~~
-
 Set which sorts of gossip connections are permitted:
-- `local` *(Boolean)* ... TODO
-- `friends` *(Boolean)* ... TODO
-- `global` *(Boolean)* ... TODO
+- `local` Enable synchronize with random local peers
+- `friends` Establish connections with peers who are friends
+- `global` Do not restrict who you connect with.
```

Comment:
I think these settings need more clarification.
Like if you set `global: true`, then does that over-ride `local: false`? your description here suggests that. I think this is more about making connections with people not on the local network. ... but then I don't know what that "friends" option is about...

Reply:
Okay, so I made some changes in relation to gossip. I had to read the code to see how the settings affected performance. 
I also changed the information about remote, but I still have doubts with the timers, I can't find any code that uses timers.connections and timers.reconnect